README.md

```diff
@@ -1,7 +1,27 @@
 # Hyperworld
+[![image](https://github.com/millennIumAMbiguity/hyperworld/assets/37588844/d5073b5f-2d76-4ef0-8276-61d05f8e8f64)](https://youtu.be/Fkyq9AxCcBs?si=RGCMwWRw0cUjR2bO)
 
 This program is a work in progress, and a lot of design decisions are up in the
 air. The code lacks a lot of needed documentation, so please contact me (possibly by
 creating an issue) if you would like help working with this source code.
 
 Instructions on how to build this project can be found in the win-x64-static directory.
+
+
+## Controls
+
+### Movement
+
+* <kbd>Left Mouse Button</kbd> and <kbd>Right Mouse Button</kbd> Forwards and backwards.
+* <kbd>Q</kbd> and <kbd>E</kbd> Roll left and right.
+* <kbd>Left Shift</kbd> Toggle between fast and slow movement.
+* <kbd>Left Ctrl</kbd> Toggle between rotate and pan mode.
+  * rotate mode: <kbd>W</kbd>, <kbd>S</kbd>, <kbd>A</kbd>, and <kbd>D</kbd> rotate up, down, left, and right.
+  * pan mode: <kbd>W</kbd>,<kbd>S</kbd>, <kbd>A</kbd>, and <kbd>D</kbd> pan up, down, left, and right.
```

Comment:
I'm not sure what the best terminology is here. I think it would make more sense to use "move", "strafe", or "truck" here instead of "pan", since panning a camera often refers to keeping the camera in a fixed position and changing which direction it's facing.

I believe it should be safe to use the same terminology for how the camera moves regardless of what Left Ctrl toggle mode the user is in. The main thing that Left Ctrl does is determine whether the camera's view gets fixed on the ideal point in front of it or whether it translates left/right/up/down without any rotation.

Reply:
strafe works